scripts/patch-release/patch-release.go

```diff
@@ -55,45 +57,37 @@ func run() error {
 		return fmt.Errorf("open local repo: %w", err)
 	}
 
-	currentBranch, err := repo.CurrentBranch()
-	if err != nil {
-		return fmt.Errorf("get current branch: %w", err)
-	}
-	logrus.Infof("Using current branch: %s", currentBranch)
-
-	newVersion, err := incVersion(version.Version)
-	if err != nil {
-		return fmt.Errorf("increment version: %w", err)
-	}
-
-	logrus.Infof("Using new version: %s", util.SemverToTagString(newVersion))
-
-	if err := updateVersionAndCreatePR(
-		repo, newVersion, currentBranch, org, remote,
-	); err != nil {
-		return fmt.Errorf("update version in local repository: %w", err)
+	currentReleaseBranch := branchPrefix + version.Version
+	for _, version := range releaseMinorVersions {
+		sv, err := convertStringToSemver(version)
```

Comment:
The value of `sv` is now just `1.29`, `1.28` or `1.27`, which is not correct. We need the current value of `version.Version` for each release branch and increment that.

Reply:
Added the changes as requested.